### PR TITLE
chinese

### PR DESCRIPTION
**Pull request by xinjiajuan**
大哥，没问题就通过我的翻译吧

### CODE REVIEW

File: res/values-zh-rCN/custom_strings.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+<resources>
+
+    <!-- Settings title to show Google Now at -1 screen on launcher. [CHAR LIMIT=50] -->
+    <string name="title_show_google_app">显示谷歌应用程序</string>
+    <!-- Settings message explaining when the -1 screen is available on an LTR device. [CHAR LIMIT=100] -->
+    <string name="msg_minus_one_on_left">当你从主屏幕向左滑动时</string>
+    <!-- Settings message explaining when the -1 screen is available on an RTL device. [CHAR LIMIT=100] -->
+    <string name="msg_minus_one_on_right">当你从主屏幕向右滑动时</string>
+    <string name="pref_show_google_now_summary" translatable="false">@string/msg_minus_one_on_left</string>
```
Comment (idoybh) [inline res/values-zh-rCN/custom_strings.xml]:
```suggestion
```

File: res/values-zh-rCN/custom_strings.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    <!-- SmartSpace -->
+    <string name="smartspace_preferences" translatable="false">Preferences</string>
```
Comment (idoybh) [inline res/values-zh-rCN/custom_strings.xml]:
```suggestion
```

File: res/values-zh-rCN/custom_strings.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    <string name="desktop_show_qsb_summary">在主屏幕上显示“一目了然”</string>
+
+    <!-- Hide labels -->
+    <string name="desktop_show_labels">在桌面上显示图标标签</string>
+    <string name="drawer_show_labels">在抽屉中显示图标标签</string>
+    <string name="show_labels_landscape">在横向模式下显示图标标签</string>
+
+    <!-- Folder titles -->
+    <string name="google_folder_title" translatable="false">Google</string>
+
```
Comment (idoybh) [inline res/values-zh-rCN/custom_strings.xml]:
```suggestion
```

File: res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -144,4 +144,8 @@
     <string name="work_apps_paused_content_description" msgid="7553586952985486433">"工作资料已被暂停。现在，工作应用无法向您发送通知、不能耗用电池电量，也无法获取您的位置信息"</string>
     <string name="work_switch_tip" msgid="808075064383839144">"暂停工作应用及其通知"</string>
     <string name="remote_action_failed" msgid="1383965239183576790">"失败：<xliff:g id="WHAT">%1$s</xliff:g>"</string>
+    <!-- Icon packs -->
+    <string name="iconPackPackageDefault" translatable="false">""</string>
```
Comment (idoybh) [inline res/values-zh-rCN/strings.xml]:
```suggestion
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
Stop touching `translatable="false"` strings

File: res/values-zh-rCN/custom_strings.xml (hunk 1 above)
Comment (xinjiajuan) [reply res/values-zh-rCN/custom_strings.xml]: I didn't change the content of the translatable="false" tag, I just copied it to compare with the original values

**Review by ALKALiKong233 — COMMENTED** [review]
Everything fine 👌

**Pull request merged**